ddl/check.lua

```diff
@@ -730,12 +730,17 @@ local function is_callable(object)
 end
 
 local function check_sharding_func_name(sharding_func_name)
+    local vshard_func_support = {
```

Comment:
It's a bit weird to allocate the table on each check.

Reply:
Fair enough, fixed.